crypto_kem/firesaber/META.yml

```diff
@@ -0,0 +1,13 @@
+name: FireSaber 
+type: kem
+claimed-nist-level: 5
+claimed-security: IND-CCA2
+length-public-key: 1312
+length-ciphertext: 1472
+length-secret-key: 3040
+length-shared-secret: 32
+nistkat-sha256: 937d9b2e139112e13d4093a6afe715deff476e4d578208b9e8e1809de43835cd
+principal-submitter: Jan-Pieter D'Anvers, Angshuman Karmakar, Sujoy Sinha Roy, Frederik Vercauteren
```

Comment:
We should probably turn this into a list. We talked about allowing either a list or a string, but now I thought about it a bit more, I don't think we should go in that direction:

* Singleton lists are not that annoying to write or process
* Allowing both `str` and `list` type there is annoying to check
* More importantly: it's very annoying if you are parsing the `META.yml` and need to detect the type of the element instead of just wrapping it in `', '.join(meta['principal-submitters'])`.

Might be best to do that in a separate PR though, because it will touch all schemes.

Reply:
#194

Reply:
PR that fixes this #195